### ISSUE TITLE
Radio Button - Selected+Focus State

### ISSUE THREAD

**Issue by bobwise** [sparkdesignsystem/spark-design-system#3118]
## Describe The Bug
When using the Radio Button component **with a keyboard** in any framework, it is unclear when the component has focus.

## To Reproduce
Without Spark, it is clear which radio group has focus:
![image](https://user-images.githubusercontent.com/1424560/80526446-be40b980-8960-11ea-84de-4bf5eb98ba7f.png)

With Spark, there is no focus styling applied and it is unclear which component has focus:
![image](https://user-images.githubusercontent.com/1424560/80526482-d0225c80-8960-11ea-8481-42aad9fb9dd2.png)

## Expected Behavior
The radio button should have a focus outline, similar to how we handle this with Checkboxes.

![image](https://user-images.githubusercontent.com/1424560/80526593-f811c000-8960-11ea-8979-42ef4fe816f1.png)
